kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -176,6 +176,10 @@ public void put(Collection<SinkRecord> records) {
           TableWriterBuilder tableWriterBuilder;
           if (config.getList(config.ENABLE_BATCH_CONFIG).contains(record.topic())) {
             String gcsBlobName = record.topic() + "_" + uuid + "_" + Instant.now().toEpochMilli();
+            String gcsFolderName = config.getString(config.GCS_FOLDER_NAME_CONFIG);
+            if (!gcsFolderName.equals("")) {
```

Comment:
For safety, `"".equals(gcsFolderName)`

Reply:
Thanks for catching this! Updated accordingly.

Reply:
Actually, thinking on this more. My suggestion was not quite correct. I suspect config.getString() returns null if not set, and thus this if check would return true if it's null. Probably want != null

Reply:
Thanks for pointing this out! The `gcsFolderName` config is set to be "" by default, but I added a `!= null` check to be safe :)